packages/clerk-js/src/core/clerk.ts

```diff
@@ -1348,15 +1350,16 @@ export class Clerk implements ClerkInterface {
       return;
     }
 
-    const tracker = createBeforeUnloadTracker(this.#options.standardBrowser);
-    const defaultRedirectUrlComplete = this.client?.signUp ? this.buildAfterSignUpUrl() : this.buildAfterSignInUrl();
```

Comment:
This default was previously placed when we thought of exposing this "navigateToTask" as a public API that could be called from custom flows.